src/pages/jonghoLee/LoginPage/LoginForm.scss

```
@@ -0,0 +1,109 @@
+@import "../common.scss";
+@import "../reset.css";
+
+.login_form {
+  @extend %flex-center;
+  flex-direction: column;
+
+  width: 348px;
+  height: 211px;
+
+  .login_id_div,
+  .login_pw_div {
+    width: 268px;
+    height: 36px;
+
+    margin: 5px auto;
+
+    .login_id_span,
+    .login_pw_span {
+      position: absolute;
+      width: 267px;
+      height: 36px;
+
+      margin-top: 5px;
+      margin-left: 1px;
```

```suggestion
      margin: 5px 0px 0px 1px;
```
- 개인차가 있겠지만 어떤 경우에만 분리해서 작성하기 보다는 한 줄로 통일 시키는 것이 보기 좋을 수 있을 것 같습니다.